src/__tests__/reducers/wordsReducer.test.js

```diff
@@ -0,0 +1,109 @@
+import { sortBy, } from 'lodash'
+import wordsReducer from '../../redux/reducers/wordsReducer'
+import {
+  fakeData,
+  DELETE_WORD,
+  EDIT_WORD,
+  EDIT_WORDS_LIST,
+  DELETE_WORDS_LIST,
+} from '../../constants'
+
+const sortedWordsList = sortBy(fakeData, data => data.word)
+
+describe('INITIAL_STATE', () => {
+  test('is correct', () => {
+    const action = { type: 'NON_EXISTENT_ACTION', }
+    const initialState = {
+      wordsList: sortedWordsList,
+    }
+
+    expect(wordsReducer(undefined, action)).toStrictEqual(initialState)
+  })
+})
+
+describe('DELETE_WORD', () => {
+  test('returns the correct state after DELETE_WORD action', () => {
```

Comment:
Please be more descriptive
E.g. `should delete smth from... `, `adds the new item...`